tests/install_upgrade_operators/csv/test_subscription_channels.py

```diff
@@ -4,8 +4,8 @@
 
 
 @pytest.mark.polarion("CNV-7169")
-def test_channels_in_manifest(kubevirt_package_manifest_channels):
-    expected_channels = {"stable", "candidate", "dev-preview"}
+def test_channels_in_manifest(kubevirt_package_manifest_channels, kubevirt_package_manifest_current_channel):
+    expected_channels = {"stable", "dev-preview", kubevirt_package_manifest_current_channel.name}
```

Comment:
`kubevirt_package_manifest_current_channel` already checks that the channel is in `kubevirt_package_manifest_channels`, why do we need to add another check here?